### DISCUSSION TITLE
Nested Albums - Albums inside albums!

### DISCUSSION THREAD

**Discussion by Vxrtiger** [ente-io/ente#782] [Enhancements]
(Don’t know if it’s been requested already)

Being able to have multiple albums inside a main one would be really helpful and keep things more organized. (Example in attachment)
![IMG_3263](https://github.com/ente-io/photos-app/assets/161477173/924fdc09-24cd-4de3-910f-b0f4c3786049)
![IMG_3265](https://github.com/ente-io/photos-app/assets/161477173/6e60eab6-b1ac-44e1-a539-2ba5d368f7ee)

**Comment by subcidr** [comment]
Much agreed! Recursive folders/albums is a very important feature for a lot of people. Really hope this could be implemented 🙌

**Comment by Kaiantics** [comment]
I came from using Samsung photos and I just subscribed to Enty Photo. I loved grouping "albums within albums" as this is a feature I used a lot. Would love to see this on the roadmap!

**Comment by Kaiantics** [comment]
> I came from using Samsung photos and I just subscribed to Enty Photo. I loved grouping "albums within albums" as this is a feature I used a lot. Would love to see this on the roadmap!

My fault! I see now it has already been added to the roadmap under "No Status." Awesome!

**Comment by vishnukvmd** [comment]
Github doesn't allow us to merge discussions, so closing this in favor of the original request here: https://github.com/ente-io/ente/discussions/928

**Comment by cykl** [comment]
It would be really useful for sync folders too. 

I mostly use ente to make some large photo folders available on mobile (master copy is a folder on a PC that should be mirrored on ente). Recursive collections could allow to:

* Share collections from an arbitrary level rather than having to share every collections.
* Easily delete all collections that have been created by shared folder.
* Easily navigate through collections and distinguish them by source folder.

**Comment by KestrelWild** [comment]
I would really really love this feature as well :)
